.github/workflows/nodejs.yml

```diff
@@ -15,10 +15,10 @@ jobs:
     strategy:
       matrix:
         config:
-          - { os: macos-11, arch: x64 }
+          - { os: macos-13, arch: x64 }
           - { os: ubuntu-latest, arch: x64 }
```

Comment:
might be worth pinning this. seems like Ubuntu 24 is working so maybe it's fine to set to that?